src/pages/case-studies/[...paths].tsx

```diff
@@ -0,0 +1,124 @@
+import type { GetStaticPaths, GetStaticPathsResult, GetStaticProps } from 'next';
```

Comment:
This file seems to be a duplicate of `src/pages/[...paths].tsx`.

Reply:
It is indeed two different files allowing to manage different page layouts (a default page for static pages, one for case studies).